test/infrastructure/clusters/test-infra/git-stats-exporter/git-stats-exporter.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: v1
```

Comment:
Thoughts on upstreaming this into KIT? Could be a useful feature

Reply:
we're shoving a lot of stuff in KIT :D  but it may make sense at some point. But let's try it out here first.